test_dagmc_stats.py

```diff
@@ -99,3 +90,33 @@ def test_get_surfaces_per_volume():
     for eh in range(known_volumes.size()):
         surfs = my_core.get_child_meshsets(known_volumes[eh]).size()
         assert(surfs == s_p_v_data[eh])
+
+def test_get_triangle_aspect_ratio():
+    """
+    Tests part of the get_triangle_aspect_ratio function
+    """
+    test_input2 = "single-cube.h5m"
+    my_core = core.Core()
+    my_core.load_file(test_input2)
+    root_set = my_core.get_root_set()
+
+    assert(abs(dagmc_stats.get_triangle_aspect_ratio(my_core, root_set) - 1000*np.sqrt(2)/400/(10-5*np.sqrt(2))) < 0.01)
```

Comment:
Doesn't this call to `get_triangle_aspect_ratio` return a list of many aspect ratios?  I'm guessing this single cube model has 12 triangles.  If so, I'm not sure exactly how to interpret this comparison.  It is perhaps coincidence that ll 12 triangles have the same aspect ratio?

Also, it is good practice to document a test in this form:
```
exp = 1000*np.sqrt(2)/400/(10-5*np.sqrt(2))) < 0.01
obs = dagmc_stats.get_triangle_aspect_ratio(my_core, root_set)
assertAlmostEqual(exp,obs)
```

Notes:
* `exp` refers to the expected result
* `obs` refers to the observed result
* `assertAlmostEqual` is a method in `nose` (I think) that allows you to handle floating point comparison that may not be exact.

Similar improvements could be made in the triangle area test below.

Reply:
I have edited the test file and please have a look again. Thank you!

For the single cube, it has 12 identical triangles with the same aspect ratio. Since model with various aspect ratio is hard to test, I choose to run test on the single cube to see if it computes the correct aspect ratio for the triangle.

Thanks for all the comments. I learned a lot from them.